frontend/src/types.ts

```diff
@@ -633,6 +633,8 @@ export enum PropertyOperator {
     NotBetween = 'not_between',
     Minimum = 'min',
     Maximum = 'max',
+    In = 'in',
```

Comment:
dumb question but can't we reuse `IContains`/`NotIContains` here by putting them in the context of `PropertyFilterType.Cohort`? though also ok with current approach if you think it's cleaner :)

Reply:
Isn't IContains a case insensitive string contains function? Seems like an overload.